packages/terra-table/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-table",
+  "main": "lib/index.js",
+  "version": "4.36.0",
+  "description": "The terra-table component provides user a way to render selectable data in a tabular format",
```

Comment:
I don't think that the description should mention selectable data since this is the Table component.  That would be the purpose of a grid not a table.

Reply:
Updated the description.